ansible/roles/nginx/tasks/main.yml

```diff
@@ -0,0 +1,9 @@
+- name: Install Nginx
+  ansible.builtin.command: sudo amazon-linux-extras install nginx1
```

Comment:
why this is used ?

Reply:
To acess the latest version provided by Amazon